client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -19,15 +20,44 @@ import { getSelectedSiteId } from 'state/ui/selectors';
 import formatCurrency from 'lib/format-currency';
 import QuerySimplePayments from 'components/data/query-simple-payments';
 import QueryMedia from 'components/data/query-media';
+import QuerySitePlans from 'components/data/query-site-plans';
+import { hasFeature, getSitePlanSlug } from 'state/sites/plans/selectors';
+import { FEATURE_SIMPLE_PAYMENTS } from 'lib/plans/constants';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const { translate, productId, product, siteId } = this.props;
+		const {
+			translate,
+			productId,
+			product,
+			siteId,
+			shouldQuerySitePlans,
+			planHasSimplePaymentsFeature,
+		} = this.props;
 
 		if ( ! product ) {
 			return <QuerySimplePayments siteId={ siteId } productId={ productId } />;
 		}
 
+		if ( shouldQuerySitePlans ) {
+			return <QuerySitePlans siteId={ siteId } />;
+		}
+
+		if ( ! shouldQuerySitePlans && ! planHasSimplePaymentsFeature ) {
+			return (
+				<div className="wpview-content wpview-type-simple-payments">
+					<div className="wpview-type-simple-payments__unsupported">
+						<div className="wpview-type-simple-payments__unsupported-icon">
+							<Gridicon icon="cross" />
+						</div>
+						<p className="wpview-type-simple-payments__unsupported-message">
+							{ translate( "Your plan doesn't include Simple Payments." ) }
```

Comment:
As @vindl suggested, let's maybe link to the plans page with the feature selected: for simple sites/atomic
<img width="902" alt="screen shot 2018-07-19 at 4 19 06 pm" src="https://user-images.githubusercontent.com/1270189/42975186-b60fbf0e-8b6f-11e8-8f75-70df24cb7fbb.png">
 this is `/plans/{site}?feature=simple-payments`

For Jetpack sites let's leave off the query param, monetization is listed under the word ads feature and doesn't align with the top plan 🤷‍♀️ 
<img width="869" alt="screen shot 2018-07-19 at 4 14 14 pm" src="https://user-images.githubusercontent.com/1270189/42975152-885c8bd2-8b6f-11e8-8b6a-118dbe901030.png">